hasura/migrations/default/1646251832285_create_table_public_distributions/up.sql

```diff
@@ -1,13 +1,13 @@
 CREATE TABLE "public"."distributions"
 (
-  "id" bigint NOT NULL,
+  "id" bigserial NOT NULL,
```

Comment:
The migrations are incremental, you will need to do a `ALTER` table if you are modifying an existing table otherwise the change won't be applied to the DB. You can use the hasura console to edit the tables, it should auto generate the migration sql files for you. Then you can do a `migration squash --from <timestamp>` if needed

Reply:
oh right, good point. the original versions of these have already been run in staging